app/src/main/java/app/foodfinderapp/MainActivity.kt

```diff
@@ -34,16 +34,24 @@ class MainActivity : AppCompatActivity() {
         }
     }
 
+    /**
+     * Initialize the contents of the MainActivity's menu
+     *
+     * @param menu The options menu in which you place your items.
+     * @return Returns true for the menu to be displayed; returns false if it will not be shown.
+     */
     override fun onCreateOptionsMenu(menu: Menu): Boolean {
-        // Inflate the menu; this adds items to the action bar if it is present.
```

Comment:
KDoc is preferred over inline comments.

Inline comments often indicate that code is not written in a readable way.